langstream-cli/src/main/java/ai/langstream/cli/commands/docker/LocalRunApplicationCmd.java

```diff
@@ -148,7 +148,8 @@ public class LocalRunApplicationCmd extends BaseDockerCmd {
 
     @CommandLine.Option(
             names = {"--langstream-runtime-docker-image"},
-            description = "Docker image of the LangStream runtime to use")
+            description = "Docker image of the LangStream runtime to use",
```

Comment:
There is also the python command that processes requirements.txt that uses a docker image